src/scenic/simulators/Gazebo_sawyer/README.md

```diff
@@ -0,0 +1,71 @@
+﻿# Scenic-Sawyer Interface
+This repo contains the code to the Scenic-Gazebo interface applied to Sawyer robot.
+## Requirements & Installation
+
+ - Scenic 3 is required, as well as Python 3.8+, ROS Noetic, and Ubuntu 20.04.
+ - The Ubuntu 20.04 requirement is due to ROS Noetic officicially supporting it. It should not be strictly required if you can get ROS Noetic running with Gazebo on your machine. Though this interface is only tested on Ubuntu 20.04.
+ - Other than the official installation, ROS Noetic can also be installed with [RoboStack](https://robostack.github.io/index.html), which can be helpful if you are on MacOS or Windows.
+ - Before you install, it might be good to consider installing Scenic outside any virtual environment for Gazebo/ROS purposes since Gazebo/ROS utilizes some Ubuntu native packages that is difficult to get from conda/pip
+ - When installing Scenic, please do the "Repository" install outlined in this page: https://scenic-lang.readthedocs.io/en/latest/quickstart.html#installation 
+ - For the Sawyer simulator, please install according to the instruction here: https://github.com/RethinkRobotics/sawyer_simulator
```

Comment:
This link was unclear on where to find the installation instructions. I ended up using the installation instructions from the tutorial: https://support.rethinkrobotics.com/support/solutions/articles/80000980381-gazebo-tutorial